dev_tools/utils/nxdl_utils.py

```diff
@@ -0,0 +1,854 @@
+# pylint: disable=too-many-lines
```

Comment:
This file has the same issues as mentioned in in-depth review performed in https://github.com/nexusformat/definitions/pull/1303/files#diff-42efc233d0fece860f874809d0c5b425905ff3341504af645d3f205b628cb5a4